### ISSUE TITLE
A11y (Accessibility) warnings when using Dropdown in SvelteKit

### ISSUE THREAD

**Issue by chrisvire** [saadeghi/daisyui#1285]
I am using DaisyUI in a [Svelte](https://svelte.dev) + [SvelteKit](https://kit.svelte.dev) application.  Recently, I [upgraded dependencies](https://github.com/sillsdev/appbuilder-pwa/pull/63) and started getting a11y (accessibility) warnings with our Dropdown component.

https://github.com/sillsdev/appbuilder-pwa/blob/upgrade-svelte-kit-525/src/lib/components/Dropdown.svelte

```<!--
@component
A simple dropdown menu from DaisyUI.
-->
<div class="dy-dropdown">
    <label tabindex="0" class="dy-btn dy-btn-ghost p-0.5 dy-no-animation">
        <slot name="label" />
    </label>
    <div tabindex="0" class="dy-dropdown-content dy-menu dy-shadow bg-base-100">
        <slot name="content" />
    </div>
</div>
```

Here is the warning that I am getting:

> [15:27:44 Svelte] 3:27:44 PM [vite-plugin-svelte] /Users/hubbard/codereview/appbuilder-pwa/src/lib/components/Dropdown.svelte:7:4 A11y: noninteractive element cannot have positive tabIndex value
> [15:27:44 Svelte] 3:27:44 PM [vite-plugin-svelte] /Users/hubbard/codereview/appbuilder-pwa/src/lib/components/Dropdown.svelte:10:4 A11y: noninteractive element cannot have positive tabIndex value

This seems to be: 
https://svelte.dev/docs#accessibility-warnings-a11y-no-noninteractive-tabindex

We followed the documentation for creating a dropdown:
https://daisyui.com/components/dropdown/

> We use a <label tabindex="0"> instead of a <button> because Safari has [a bug](https://bugs.webkit.org/show_bug.cgi?id=22261) that prevents the button from being focused.
> Using tabindex="0" is required so the dropdown can be focused.

What is the proper way to have a dropdown with DaisyUI that doesn't break accessibility?
